portfolio/src/main/webapp/script.js

```diff
@@ -79,13 +79,18 @@ function createCommentDiv(commentObj) {
 
   const nameElement = document.createElement('h4');
   nameElement.innerText = commentObj.name;
+  nameElement.className = "tooltip";
+  const tooltipElement = document.createElement('span');
+  tooltipElement.className = "tooltiptext";
+  tooltipElement.innerText = commentObj.time;
+  nameElement.appendChild(tooltipElement);
   commentDiv.appendChild(nameElement);
 
   const contentElement = document.createElement('p');
   contentElement.innerText = commentObj.content;
   commentDiv.appendChild(contentElement);
 
-  // TODO: do something useful with the time information
+  // TODO: Convert from fairly useless MS time to hour/minute.
```

Comment:
Haha was just about to put a comment above with this suggestion! Could be good to include the date as well as the hour/minute (plus timezone (feel free to just use eastern time to make it easier))

Reply:
I definitely would like to. I looked into it a little bit and it seems like there's no good library function to convert, so I would have to write a conversion function myself. I might add it after I've made some progress on the week 4 walkthrough.

Reply:
Sounds good!